pkg/deploy/assets/aks-development.json

```diff
@@ -265,10 +265,6 @@
             "metadata": {
                 "description": "Specifies the max graceful termination time interval in seconds for the auto-scaler of the AKS cluster."
             }
-        },
-        "vpnCACertificate": {
```

Comment:
The `gatewaySubnetName` and `gatewaySubnetPrefix` can also be removed, as well as the subnet definition:
```json
{
    "name": "[parameters('gatewaySubnetName')]",
    "properties": {
        "addressPrefix": "[parameters('gatewaySubnetPrefix')]"
    }
}